doc/null_safety/null_safe_migration.md

```diff
@@ -188,6 +196,21 @@ flowchart TD
   End_Required[/"Make it <strong>required</strong>\n<code>late SomeType propName;</code>"\]
 ```
 
+### Wrapper and `connect`ed components and required props
+
+There may be some cases where you have a wrapper component or `connect`ed component that sets some required props, 
+but you get lints and runtime errors about missing props when consuming them.
+
+See [this section](../null_safety_and_required_props.md#disabling-required-prop-validation-for-certain-props)
+of the null safety and required props docs for instructions on how to handle these cases and suppress this validation.
+
+#### connect
+
+We'll be adding a codemod to help migrate the `connect` case: https://github.com/Workiva/over_react_codemod/issues/295
+
+Alternatively, you could refactor your component to instead utilize [OverReact Redux hooks](./over_react_redux_documentation.md#hooks), 
```

Comment:
This link doesn't work - I think it needs more `../` to get out of the null_safety directory or something?